Sources/Commands/SwiftBuildTool.swift

```diff
@@ -43,14 +43,8 @@ public class SwiftBuildTool: SwiftTool<BuildToolOptions> {
             try build(yamlPath: yaml, target: options.buildTests ? "test" : nil)
 
         case .clean:
-            if options.enableNewResolver {
-                try getActiveWorkspace().clean()
-            } else {
-                // FIXME: This test is lame, `removeFileTree` shouldn't error on this.
-                if exists(buildPath) {
-                    try removeFileTree(buildPath)
-                }
-            }
+            print("warning: swift build --clean is deprecated. Use 'swift package clean' instead.")
```

Comment:
Let's adopt a practice of including the SR # in the diagnostic when we deprecate things.